CONTRIBUTING.rst

```diff
@@ -0,0 +1,173 @@
+have this installed as ``virtualenv`` or ``pyvenv``. If you do not, 
+this can be done with ``pip`` easily:
+
+::
+
+    $ pip install virtualenv
+
+Once you have ``virtualenv`` installed, just fire up a shell and create
+your own environment:
+
+::
+
+    $ virtualenv ENV
+
+Where ``ENV`` is a directory to place the new virtual environment.
+
+Now, whenever you want to work on the project, you only have to activate 
+the corresponding environment.
+
+    On **Unix based** systems (OS X and Linux), this can be done with:
+
+    ::
+
+        $ source ENV/bin/activate
+
+    And on **Windows** this is done with:
+
+    ::
+
+        $ ENV\scripts\activate
+
+Note that if this works successfuly, your terminal prompt should look like:
+
+::
+
+    (ENV)$
+
+Finally, you should install vulture inside the activated virtualenv.
+
+In order to deactivate the ``virtualenv``, simply use
+
+::
+
+    (ENV)$ deactivate
+
+Installation
+------------
+
+There are two ways of installing vulture. By installing releases from PyPI, 
+or directly from source.
+
+PyPI Installation
+~~~~~~~~~~~~~~~~~
+
+::
+
+  $ pip install vulture
+
+Source installation
+~~~~~~~~~~~~~~~~~~~
+
+::
+
+  $ git clone https://github.com/jendrikseipp/vulture
+  $ cd vulture
+  $ pip install .
+
+Similarly, you can install your own fork of vulture.
+
+Testing
+-------
+
+Vulture uses tox for testing. You can read more about
+it at the `tox documentation <https://tox.readthedocs.io>`_.
+
+To install ``tox``, run:
+
+::
+
+  $ pip install tox
+
+To run ``tox``, simply use:
+
+::
+
+  $ tox
+
+Coding Standards
+----------------
+
+Creating a new branch
+~~~~~~~~~~~~~~~~~~~~~
+
+To start working on a pull request, you first need to create a new branch 
+where you will work. Do not change files when you are on your fork’s master 
+branch. The reason why you should never develop on your master branch is 
+because your fork’s master branch should always be synchronized with the 
+main repository’s master branch, which is much more challenging if it has 
+new commits on it. This is why we create our own branch:
+
+::
+
+  $ git checkout -b your-new-branch
+
+Naming the branches
+~~~~~~~~~~~~~~~~~~~
+
+A branch name should be descriptive about the feature/issue it is working on, 
+but at the same time, be a small string. Be careful not to re-use existing 
+branch names.
+
+Commits
+~~~~~~~
+
+Each commit should be atomic and only do the changes specified in it's messages.
+
+Commit messages are the way people in future, reading the code, can 
+understand changes. Make sure that they describe the changes in a clear and 
+crisp manner.
+
+Pull Request
+------------
+
+How to send a Pull Request?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To do this, you will have to go on GitHub, on your fork page. You should 
+change the branch to the one you have worked on and submitted the commit on. 
+Now you can create a Pull Request by clicking the ``New Pull Request`` button 
+in the pull request tab.
```

Comment:
~~To do this, you will have to go to~~ Visit your fork on Github, change the branch to the one you have ~~worked on and submitted the commit~~ committed on. Now you can create a Pull Request by clicking the New Pull Request button ~~in the pull request tab~~.